src/main/java/seedu/storemando/model/item/QuantityComparator.java

```diff
@@ -0,0 +1,17 @@
+package seedu.storemando.model.item;
+
+import java.util.Comparator;
+
+public class QuantityComparator implements Comparator<Item> {
```

Comment:
Should this class be called QuantityComparator? You are comparing between 2 items and not their quantity. I suggest the class name should be changed to ItemComparator.